src/platforms/common/migration/index.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Migration
+description: "Migrating between versions of Sentry SDK for .NET."
+sidebar_order: 10
+---
+
+<Note>
+
+Migrating to `Sentry` version 3.0.0
+
+</Note>
+
+# Breaking changes
+
+Please read through **all** breaking changes relevant to you before upgrading.
+
+## !Important: Grouping Changes
+
+The format of the stack traces were greatly improved with the help of Ben Adam's iconic library `Ben.Demystifier`. Frames now include details such as `async`, `static`, the actual return type, method parameters, tuples and their names, and more.
+
+One big caveat is that this will **affect grouping!**. If you prefer to stay with the original stack trace format, you can opt-out of this feature with:
+
+```
+options.StackTraceMode = StackTraceMode.Original;
+```
+
+## ASP.NET
+
+ASP.NET (**not Core**) users need to install an additional package:
+
+`Sentry.AspNet`.
+
+Without this, HTTP related information will not be added to events.
+
+You can plug it in to the `Init` as follows:
+```
+SentrySdk.Init(o =>
+{
+    o.AddAspNet();
+});
+```
+
+The motivation was to remove the reference to `System.Web` to decouple the core of the package with any specific namespaces of the .NET Framework, which improves the experience from other environments such as game engines like Unity and [Godot](https://gatomalo.dev/blog/2020/03/21/error-monitoring-godot-sentry/#android).
+
+## ASP.NET Core Environment Casing
+
+To match Sentry's default environment naming convention, unless explicitly set otherwise, the Sentry environment will be reported lowercase. That means if `ASPNET_ENVIRONMENT` is equal to `Production`, the Sentry SDK will report it as `production` #551
```

Comment:
```suggestion
To match Sentry's default environment naming convention, unless explicitly set otherwise, the Sentry environment will be reported lowercase. That means if `ASPNET_ENVIRONMENT` is equal to `Production`, the Sentry SDK will report it as `production`.
```